metamusic/clients/musicbrainz.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+"""
+Author : Salvador <salvador.estrella.ortiz@gmail.com>
+Date   : 2022-09-30
+Purpose: Query Metadata from MusicBrainz API
+"""
+
+import argparse
+import requests
+from pprint import pprint
+from time import sleep
+
+
+# --------------------------------------------------
+def get_args():
+    """Get command-line arguments"""
+
+    parser = argparse.ArgumentParser(
+        description="Retrieve metadata from MusicBrainz API",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "-a",
+        "--artist",
+        help="An artist to search for",
+        metavar="str",
+        type=str,
+        required=True,
+    )
+
+    return parser.parse_args()
+
+
+# --------------------------------------------------
+class MusicMetadata:
+    artist_endpoit = "http://musicbrainz.org/ws/2/artist/"
+    release_groups_endpoint = "http://musicbrainz.org/ws/2/release-group/"
+    releases_endpoint = "http://musicbrainz.org/ws/2/release/"
+
+    @classmethod
+    def _pick_artist(cls, artist_list):
+        for artist in artist_list:
+            if artist["score"] == 100:
+                return artist
+
+    @classmethod
+    def get_artist_info(cls, artist: str) -> dict:
+        endpoint = cls.artist_endpoit
+        artist = artist.split()
+        artist = "%20".join(artist)
+        artist_query = f"{endpoint}?query={artist}&limit=3&fmt=json"
```

Comment:
**suggestion:** Don't define a variable if used only once.

```suggestion
        artist = artist.split()
        artist = "%20".join(artist)
        artist_query = f"{cls.artist_endpoit}?query={artist}&limit=3&fmt=json"
```

Reply:
You're right, fixed.